hazelcast-client/src/main/java/com/hazelcast/client/config/ClientNetworkConfig.java

```diff
@@ -299,4 +303,67 @@ public ClientNetworkConfig setAwsConfig(ClientAwsConfig clientAwsConfig) {
     public ClientAwsConfig getAwsConfig() {
         return clientAwsConfig;
     }
+
+
+    /**
+     * Returns the outbound port definitions. It is possible that null is returned if not defined.
+     * @return list of outbound port definitions
+     */
+    public Collection<String> getOutboundPortDefinitions() {
+        return outboundPortDefinitions;
+    }
+
+    /**
+     * Returns the outbound ports. It is possible that null is returned if not defined.
+     * @return list of outbound ports
+     */
+    public Collection<Integer> getOutboundPorts() {
+        return outboundPorts;
+    }
+
+    /**
+     * Set outbound port definitions
+     * @param outboundPortDefinitions outbound port definitions
+     * @return ClientNetworkConfig
+     */
+    public ClientNetworkConfig setOutboundPortDefinitions(final Collection<String> outboundPortDefinitions) {
+        this.outboundPortDefinitions = outboundPortDefinitions;
+        return this;
+    }
+
+    /**
+     * Set outbond ports
+     * @param outboundPorts outbound ports
+     * @return ClientNetworkConfig
+     */
+    public ClientNetworkConfig setOutboundPorts(final Collection<Integer> outboundPorts) {
+        this.outboundPorts = outboundPorts;
+        return this;
+    }
+
+    /**
+     * Add outbound port to the outbound port list
+     * @param port outbound port
+     * @return ClientNetworkConfig
+     */
+    public ClientNetworkConfig addOutboundPort(int port) {
+        if (outboundPorts == null) {
+            outboundPorts = new HashSet<Integer>();
+        }
+        outboundPorts.add(port);
+        return this;
+    }
+
+    /**
+     * Add outbound port definition to the outbound port definition list
+     * @param portDef outbound port definition
+     * @return ClientNetworkConfig
+     */
+    public ClientNetworkConfig addOutboundPortDefinition(String portDef) {
```

Comment:
@pveentjer this code is same as com.hazelcast.config.NetworkConfig#addOutboundPortDefinition. All check are done later in the ClientConnectionManagerImpl.